horace_core/sqw/PixelData/@MultipixBase/private/set_nbins_.m

```diff
@@ -0,0 +1,11 @@
+function obj = set_nbins_(obj,val)
+%SET_NBINS_ defines number of bins in each image to combine together
+% val -- single positive value, describing number of bins in
+%        the images to be combined. Single value as have to be same for all
+%        images
+if ~isnumeric(val) || val < 1
```

Comment:
Why is it not possible to have 0 bins?

Reply:
pixels are placed in at least one bin.  0 violates (unnecessary) object's integrity.